README.md

```diff
@@ -9,3 +9,681 @@ ExpressRouterAdapter is available as a package on NPM for use with a module bund
 ```shell
 npm install --save @symbiotic/express-router-adapter
 ```
+
+The core imports
+```ts
+import {
+  CompositeRouteProvider,
+  RouteProvider,
+  RouterMetaBuilder,
+  HTTPError,
+  HTTPResponse,
+  SecurityContextProvider,
+  ISecurityContext,
+  ExpressRouterAdapter
+} from '@symbiotic/express-router-adapter';
+```
+
+## Getting started
+
+ExpressRouterAdapter enhances express, so lets start with a simple express app. This is not a deep dive intro express, see express documentation.
+
+```js
+
+const app = express();
+
+app.get('/pets/:petId', (req, res) => {
+  res.json({ petId: req.params.petId, name: 'Sawyer '});
+});
+
+app.listen(4000);
+
+```
+
+Lets install ExpressRouterAdapter and dependncies
+
+```shell
+npm install --save @symbiotic/express-router-adapter aurelia-dependency-injection reflect-metadata
+```
+
+We are using IOC in our examples but you could new up all the classes yourself if you choose, in which case you would not need `aurelia-dependency-injection` and `reflect-metadata`.
+
+Now we can add ExpressRouterAdapter and refactor to use RouteMetaBuilder
+
+```js
+import 'reflect-metadata'; // polyfill for aurelia-dependency-injection
+import express from 'express';
+import { ExpressRouterAdapter, RouterMetaBuilder, RouteProvider } from '@symbiotic/express-router-adapter';
```

Comment:
Break into multiple line so it does get truncated

![image](https://user-images.githubusercontent.com/1964631/61544622-d9ff5280-aa13-11e9-99f5-abb016777ffa.png)